.gitignore

```diff
@@ -34,6 +34,14 @@ tokenizer.json
 !test_bpe_tokenizer.bin
 !test_tiktoken_tokenizer.model
 
+# AOTI temporary files
+*.cubin
+*kernel_metadata.json
+*kernel.cpp
+*wrapper_metadata.json
+*wrapper.cpp
```

Comment:
does wrapper .cpp stick around? AOTI compilation doesnt clean it up after generating the .so and .cubin?

Reply:
right now it will generating lots of files, including cubin, wrapper.cpp,etc, and will be auto cleaned up.